### DISCUSSION TITLE
Discussions Drives into December 🚗 [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#40578] [New to GitHub]
We’re in the final month of the year and for many of us that means celebrations abound - so let’s kick off the month by celebrating our members 🙇

🌟**Community Superstars**: A round of applause for these community citizens 👏
* @Preetiraj3697: For sharing Git commands to help @inswo [clone their repository](https://github.com/orgs/community/discussions/39772) - thanks for being so straightforward! 
* @BREN0sx: Came in hot with links and instructions for [connecting GitHub and Pycharm](https://github.com/orgs/community/discussions/39596) - nice work 🔥 
* @MitkoVtori: Screenshots, with clear markings helped a user create a [pull request from two different branches](https://github.com/orgs/community/discussions/39864). Appreciate you chiming in!

🆒 **Cool stuff**™️ happening around GitHub and the Community
* **GitHub Skill** 🆙: This week @ossanna16 walks us through how to [suggest changes](https://github.com/orgs/community/discussions/40348) on a pull request - don’t miss it!
* **Exciting new GitHub features powering machine learning**: @sethjuarez shares the [exciting enhancements](https://github.blog/2022-11-22-exciting-new-github-features-powering-machine-learning/) in GitHub that empower Machine Learning practitioners to do more 🤖.
* **Project Readme Code Like It’s 1995**: Go back to basics–tips on securing your OSS project, developer happiness and more in [this episode of the Project Readme podcast](https://github.com/readme/podcast/java-programming-language), 🎧hosted by @nerdneha and @martinwoodward.

🧭 **Release Radar**: The latest and greatest shipping to your account!
* **Updates to the two-factor authentication setup flow**: [Learn more](https://github.blog/changelog/2022-11-21-updates-to-the-two-factor-authentication-setup-flow/) about how we’ve changed the UI on 2FA 🗝️. 
* **Repository archive date now shown in UI**: A small, but mighty change has landed in the GitHub universe - [find the details here](https://github.blog/changelog/2022-11-23-repository-archive-date-now-shown-in-ui/). 

Happy first of the month y’all! Have any fun celebrations happening this month - I want to hear about them in the comments!

**Comment by Preetiraj3697** [comment]
Welcome 😊

**Reply by jahongirdev** [reply]
Welcome too!

**Comment by MitkoVtori** [comment]
I didn't expected to be on the list. Thank you! :)

**Reply by queenofcorgis** [reply]
You earned it! Thanks for being such a wonderful community member 🤩

**Reply by huange888** [reply]
i like it

**Comment by jahongirdev** [comment]
Hello!

**Reply by TKKKKKKKKKKKK** [reply]
圣诞快乐🎄
